README.md

```diff
@@ -1,6 +1,6 @@
 # [pyp-w2] Collections Hierarchy
 
-Today, we will re implement part of the Python collections hierarchy using Mixins.
+Today, we are going to implement part of the Python collections hierarchy using Mixins.
```

Comment:
I thought it was meant to be re-implement since we are just making new list and dict objects, so implementing new objects